vayesta/core/qemb/qemb.py

```diff
@@ -1561,6 +1589,7 @@ def _reset_fragments(self, *args, **kwargs):
     def _reset(self):
         self.e_corr = None
         self.converged = False
+        self.e_rpa = None
```

Comment:
is this ever initialised in `__init__`? If we definitely need it to be a variable in the object then make sure to initialise it rather than just assigning it later

Reply:
`__init__` actually calls `_reset`, so that's part of the function of adding it here.